packages/location-management/package.json

```diff
@@ -35,15 +36,18 @@
   "license": "Apache-2.0",
   "private": false,
   "dependencies": {
-    "@opensrp/reducer-factory": "^0.0.11"
+    "uuidv4": "^6.2.5",
+    "formik-antd": "^2.0.1",
+    "tree-model": "^1.0.7",
+    "@opensrp/reducer-factory": "^0.0.11",
+    "@opensrp/server-service": "^0.0.12",
+    "@opensrp/store": "^0.0.7"
```

Comment:
lets move this to peer dependencies if its being used in the code, otherwise devDependncies.

Reply:
addressed